iOS-389 Audiobook background download

ErnestFan opened the pull request:
**What's this do?**
Merge in feature branch into develop branch

**Why are we doing this? (w/ JIRA link if applicable)**
[iOS-389](https://jira.nypl.org/browse/IOS-389)

**How should this be tested? / Do these changes have associated tests?**
Already QA-ed

**Dependencies for merging? Releasing to production?**
NYPLAudiobookToolkit, NYPLAEToolkit and iOS-Utilities are updated

**Does this include changes that require a new SimplyE/Open eBooks build for QA?**
Yes

**Has the application documentation been updated for these changes?**
N/A

**Did someone actually run this code to verify it works?**
@ErnestFan

ErnestFan commented:
@ettore Most commits in this PR have been reviewed before, except the latest ones where I updated NYPLAudiobookToolkit, bump build number and merge in changes from `develop` branch.

ettore commented:
> @ettore Most commits in this PR have been reviewed before, except the latest ones where I updated NYPLAudiobookToolkit, bump build number and merge in changes from `develop` branch.

sounds good... in the last commit I see 2 strings for MARKETING_VERSION and CURRENT_PROJECT_VERSION for each target, that's probably a git merge fail (I often have to fix it manually myself when I merge)... can you fix it? I am guessing we want 3.9.0 (3)...

ErnestFan commented:
> > @ettore Most commits in this PR have been reviewed before, except the latest ones where I updated NYPLAudiobookToolkit, bump build number and merge in changes from `develop` branch.
> 
> sounds good... in the last commit I see 2 strings for MARKETING_VERSION and CURRENT_PROJECT_VERSION for each target, that's probably a git merge fail (I often have to fix it manually myself when I merge)... can you fix it? I am guessing we want 3.9.0 (3)...

Fixed, thanks for the catch.
